SimMuon/GEMDigitizer/src/ME0ReDigiProducer.cc

```diff
@@ -139,14 +143,18 @@ void ME0ReDigiProducer::buildDigis(const ME0DigiPreRecoCollection & input_digis,
       if (smearTiming_) newTof += CLHEP::RandGaussQ::shoot(engine, 0, timeResolution_);
 
       // arrival time in ns
-      const float t0(centralTOF_[ nPartitions_ * (detId.layer() -1) + detId.roll() - 1 ]);
+      //const float t0(centralTOF_[ nPartitions_ * (detId.layer() -1) + detId.roll() - 1 ]);
+      int index = nPartitions_ * (detId.layer() -1) + detId.roll() - 1;
+      if(detId.roll() == 0) index = nPartitions_ * (detId.layer() -1) + detId.roll();
```

Comment:
but actually roll number gives 0 at the moment, we can live as it is now, given also the time we will still use that geometry @pietverwilligen